core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -138,12 +140,12 @@ public void startDebugLogging() {
 
   public void startTracing(String applicationName) {
     if (trace) {
-      Trace.on(applicationName);
+      Trace.startSpan(applicationName, Sampler.ALWAYS);
```

Comment:
I'm not sure if "ALWAYS" is best here, or if the default is better... the default appears to disable the span if tracing is not enabled for the current thread... which seems like what we want, but I'm unsure of the implications.

Reply:
Leaving any changes to sampling to future work.